### PR TITLE
Start on product page

### PR DESCRIPTION
**Pull request by andymsuk**

### CODE REVIEW

File: styles/pages/_product.scss [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    background: $accessible-red;
+    padding: 1rem;
+    text-align: center;
+    font-family: 'Timothy';
+    text-decoration: none;
+
+    &:hover,
+    &:active,
+    &:focus {
+      // TODO - what should this be?? 🤔
```
Comment (andymsuk) [inline styles/pages/_product.scss]: @vanitabarrett - thoughts for this?

File: components/Breadcrumbs.tsx [hunk 2]
```diff
@@ -0,0 +1,33 @@
+import Link from 'next/link';
+
+import { NavLink } from "../types";
+
+interface Props {
+  links: NavLink[]
+}
+
+export default function Breadcrumbs({ links }: Props) {
```
Comment (vanitabarrett) [inline components/Breadcrumbs.tsx]: They're not all links though...

File: pages/[categoryId]/[subCategoryId]/[productSlug].tsx [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+    {
+      href: `/${categoryInfo.id}`,
+      text: categoryInfo.name
+    },
+    {
+      href: `/${categoryInfo.id}/${subCategoryInfo.id}`,
+      text: subCategoryInfo.name
+    },
+    {
+      href: undefined,
```
Comment (vanitabarrett) [inline pages/[categoryId]/[subCategoryId]/[productSlug].tsx]: Can it just not be passed in instead?

File: styles/pages/_product.scss (hunk 1 above)
Comment (vanitabarrett) [reply styles/pages/_product.scss]: Screenshot 2020-11-10 at 10 51 05

**Comment by vanitabarrett** [comment]
Product focus states could look like this:
I changed the padding on each link from `15px 10px` to `10px` so the border spacing looked more even
Screenshot 2020-11-10 at 10 53 36

(move the "New" label -5px top and left)
Screenshot 2020-11-10 at 10 54 10

File: styles/pages/_product.scss (hunk 1 above)
Comment (andymsuk) [reply styles/pages/_product.scss]: Done 👍

**Comment by andymsuk** [comment]
Updated product focus states as suggested 👍

File: pages/[categoryId]/[subCategoryId]/[productSlug].tsx [hunk 4]
```diff
@@ -40,11 +40,16 @@ export default function ProductPage({
...
+          </Link>
           <p className="product-page__description">
             {htmlParser(product.description.replace(/\n/g, '<br />'))}
           </p>
           <Link href={product.url}>
-            <a className="product-page__buy">
+            <a className="product-page__buy gel-hide-lte@s">
               Buy on Etsy
             </a>
           </Link>
```
Comment (vanitabarrett) [inline pages/[categoryId]/[subCategoryId]/[productSlug].tsx]: Lets always keep the buy button at the top

File: styles/pages/_product.scss [hunk 5]
```diff
@@ -15,13 +15,12 @@
   &__description {
     font-weight: 300;
     line-height: 1.5;
-    margin-bottom: 2rem;
+    margin: 2rem 0;
```
Comment (vanitabarrett) [inline styles/pages/_product.scss]: Should this just be in px?

Comment (andymsuk) [reply styles/pages/_product.scss]: If we don't want the spacing to change if the user changes the text size? 🤷‍♂️

Comment (andymsuk) [reply styles/pages/_product.scss]: Changed

File: pages/[categoryId]/[subCategoryId]/[productSlug].tsx (hunk 4 above)
Comment (andymsuk) [reply pages/[categoryId]/[subCategoryId]/[productSlug].tsx]: Done

**Review by vanitabarrett — APPROVED** [review]

**Pull request merged**